### DISCUSSION TITLE
support wlr-output-management-unstable-v1

### DISCUSSION THREAD

**Discussion by megatux** [YaLTeR/niri#282] [Ideas]
To make programs like [kanshi](https://git.sr.ht/~emersion/kanshi) works.

```
$kanshi
compositor doesn't support wlr-output-management-unstable-v1
```

Does it breaks the output configuration `position` option?

**Comment by YaLTeR** [comment]
The way this should be implemented is, receiving an output configuration from the wlr protocol should apply it, then if a niri config is changed, the output configuration from the config should apply only if that section of the config had changed.

**Reply by YaLTeR** [reply]
Made an issue here: https://github.com/YaLTeR/niri/issues/370
